feature/foryou/src/main/java/com/google/samples/apps/nowinandroid/feature/foryou/ForYouViewModel.kt

```diff
@@ -148,15 +160,20 @@ class ForYouViewModel @Inject constructor(
 
     val interestsSelectionState: StateFlow<ForYouInterestsSelectionUiState> =
         combine(
+            syncStatusMonitor.isSyncing,
             followedInterestsUiState,
             topicsRepository.getTopicsStream(),
             authorsRepository.getAuthorsStream(),
-            snapshotFlow { inProgressTopicSelection },
-            snapshotFlow { inProgressAuthorSelection },
-        ) { followedInterestsUserState, availableTopics, availableAuthors, inProgressTopicSelection,
-            inProgressAuthorSelection ->
+            combine(
+                snapshotFlow { inProgressTopicSelection },
+                snapshotFlow { inProgressAuthorSelection },
+                ::Pair,
+            )
+        ) { isSyncing, followedInterestsUserState, availableTopics, availableAuthors,
+            (inProgressTopicSelection, inProgressAuthorSelection) ->
 
-            when (followedInterestsUserState) {
+            if (isSyncing) ForYouInterestsSelectionUiState.Loading
```

Comment:
Nit: I find the single line `if`/`else` a bit harder to read here, over using brackets.

Reply:
Dropped as per the comment below.